static/assets/arrow-left.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Shouldn't we be using something from the icon library? like `arrow-left` or `arrow-left-circle`?

Reply:
I opted to use the svg from Jade's figma designs because the arrow left from AUI is completely different.

Top arrow is the AUI component and the second is the one from the figma designs (and what the Atlaskit arrow looks like):

<img width="61" alt="Screen Shot 2022-06-06 at 2 33 05 pm" src="https://user-images.githubusercontent.com/37155488/172095479-98d73941-91b9-446b-bf54-83e956494ac0.png">

Reply:
Are we use Jade is using the correct one? or maybe we need to update AUI?

Reply:
I had to add the same arrow to the nav in Jenkins and it's the same is what's in the figma designs for GHE.

How would we go about updating AUI?